UI/test/integration-tests/singleSegmentSpeedLimitSpec.js

```diff
@@ -10,7 +10,7 @@ define(['chai', 'testHelpers'], function(chai, testHelpers) {
     });
     expect(features.length).not.to.equal(0);
     _.each(features, function(feature) {
-      expect(feature.getProperties().value).to.equal(limitValue);
+      expect(feature.getProperties().value.value).to.equal(limitValue);
```

Comment:
Can this "value.value" be changed to something more explicit?

Reply:
Let's leave that "value.value" like it is because to switch that we will have a lot of impact on the rest of the code